requirements/edx/edx-private.txt

```diff
@@ -2,7 +2,7 @@
 
 # For Harvard courses:
 -e git+https://github.com/gsehub/xblock-mentoring.git@4d1cce78dc232d5da6ffd73817b5c490e87a6eee#egg=xblock-mentoring
--e git+https://github.com/open-craft/problem-builder.git@90b2ecd8e948f34162e558b22f60e90287385f32#egg=xblock-problem-builder
+-e git+https://github.com/open-craft/problem-builder.git@v2.0.1#egg=xblock-problem-builder
```

Comment:
Why are you using -e instead of pinning the version?  I think in general we want to be moving towards pinning the version instead of using editable mode which is meant for development.